contrib/kava-3/params.go

```diff
@@ -0,0 +1,383 @@
+// kava3 contains the suggested genesis parameters for the kava-3 mainnet.
+package kava3
+
+import (
+	"fmt"
+	"time"
+
+	tmtypes "github.com/tendermint/tendermint/types"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/genutil"
+
+	"github.com/kava-labs/kava/x/auction"
+	"github.com/kava-labs/kava/x/bep3"
+	"github.com/kava-labs/kava/x/cdp"
+	"github.com/kava-labs/kava/x/committee"
+	"github.com/kava-labs/kava/x/incentive"
+	"github.com/kava-labs/kava/x/kavadist"
+	"github.com/kava-labs/kava/x/pricefeed"
+)
+
+const (
+	kavaDenom              = "ukava"
+	bnbDenom               = "bnb"
+	usdxDenom              = "usdx"
+	referenceAsset         = "usd"
+	bnbSpotMarketID        = bnbDenom + ":" + referenceAsset
+	bnbLiquidationMarketID = bnbDenom + ":" + referenceAsset + ":" + "30"
+	debtDenom              = "debt"
+)
+
+func AddSuggestedParams(cdc *codec.Codec, genDoc tmtypes.GenesisDoc, chainID string, genesisTime time.Time) (tmtypes.GenesisDoc, error) {
+
+	// Add tendermint params
+
+	genDoc.ChainID = chainID
+	genDoc.GenesisTime = genesisTime
+
+	// Add app params
+
+	var appState genutil.AppMap
+	if err := cdc.UnmarshalJSON(genDoc.AppState, &appState); err != nil {
+		return tmtypes.GenesisDoc{}, err
+	}
+
```

Comment:
we need to add an `addAuthState` function that adds the bep3 deputy to state and populates it's account with the bnb that can be used for atomic swaps.